Bundle/CoreBundle/EventSubscriber/WidgetSubscriber.php

```diff
@@ -70,9 +70,13 @@ public function onFlush(OnFlushEventArgs $args)
                 continue;
             }
 
-            $view = $entity->getView();
-            $this->updateViewCss($view);
-            $this->updateTemplateInheritorsCss($view);
+            /** @var Widget $entity */
+            foreach ($entity->getWidgetMaps() as $widgetMap) {
+                $view = $widgetMap->getView();
```

Comment:
check the WidgetMap action (we don't want to do that if widget map action is DELETE)